srv/salt/ceph/rbd/benchmarks/create_image.sls

```diff
@@ -1,3 +1,11 @@
-create benchmark image:
+create benchmark pool:
   cmd.run:
-    - name: rbd create --size 2048 deepsea_benchmark
+    - name: ceph osd pool create {{ salt['pillar.get']('rbd_benchmark_pool', 'deepsea_rbd_benchmark') }} 256 256
+
+{% for host in salt.saltutil.runner('select.minions', cluster='ceph', roles='storage') %}
+create {{ host }} benchmark image:
+  cmd.run:
+    - name: rbd create --pool {{ salt['pillar.get']('rbd_benchmark_pool', 'deepsea_rbd_benchmark') }} --size 2048 {{ salt['pillar.get']('rbd_benchmark_image_prefix', 'rbd_bench_img_') }}{{ host }}
```

Comment:
Do we want to give the flexibility of changing the pool name?

Reply:
hmm good question. I'd certainly recommend using a more expicit pool name (maybe ```rbd_deepsea_benchmark_pool```). We could also add this feature later...should be as simple as pulling a name from the pillar.